cli/src/main/kotlin/commands/DownloaderCommand.kt

```diff
@@ -274,9 +275,30 @@ class DownloaderCommand : CliktCommand(name = "download", help = "Fetch source c
                     }
                 }
 
-                if (archiveMode == ArchiveMode.BUNDLE && archiveAll()) {
-                    packageDownloadDirs.forEach { (_, dir) ->
-                        dir.safeDeleteRecursively(baseDirectory = outputDir)
+                if (archiveMode == ArchiveMode.BUNDLE) {
+                    val pathsToExclude =
```

Comment:
As discussed, esp. this code might become much simpler if we'd make license category filter already apply to the download, not only to the archiving; and it would also yield a nice performance improvement in most cases, I guess.

Reply:
Changed the implementation to your suggestion.
The downloads are only limited if the cli is being used. Not if the downloader is being used as a library.
To make that clear I added a comment to the `reference.conf` file.